modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,293 @@
+// Module included in the following assemblies:
+//
+// * machine_management/
+// * machine_management/
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-node-custom-partition_{context}"]
+= Optional: Adding a new {op-system} worker node with a custom `/var` partition in AWS
+
+{product-title} supports partitioning devices during installation by using machine configs that are processed during the bootstrap. However, if you use `/var` partitioning, the device name must be determined at installation and cannot be changed. This means that you cannot add different instance types as nodes if they have a different device naming schema. For example, if you configured the `/var` partition with the default AWS device name for `m4.large` instances, `dev/xvdb`, you cannot directly add an AWS `m5.large` instance, as `m5.large` instances use a `/dev/nvme1n1` device by default. The device might fail to partition due to the different naming schema.   
```

Comment:
My translation sense wonders if using a `/var` as an adjective for some noun might be better here. 

```suggestion
{product-title} supports partitioning devices during installation by using machine configs that are processed during the bootstrap. However, if you use `/var` [$NOUN] partitioning, the device name must be determined at installation and cannot be changed. You cannot add different instance types as nodes if they have a different device naming schema. For example, if you configured the `/var` partition with the default AWS device name for `m4.large` instances, `dev/xvdb`, you cannot directly add an AWS `m5.large` instance, as `m5.large` instances use a `/dev/nvme1n1` device by default. The device might fail to partition due to the different naming schema.   
```